lib/migration.c

```diff
@@ -534,6 +534,60 @@ migration_region_access(vfu_ctx_t *vfu_ctx, char *buf, size_t count,
     return count;
 }
 
+bool
+migration_feature_supported(uint32_t flags) {
+    switch (flags & VFIO_DEVICE_FEATURE_MASK) {
+        case VFIO_DEVICE_FEATURE_MIGRATION:
+            return !(flags & VFIO_DEVICE_FEATURE_SET); // not supported for set
+        case VFIO_DEVICE_FEATURE_MIG_DEVICE_STATE:
```

Comment:
what does it mean to set this flag.

Reply:
If we're trying to call `SET` on `VFIO_DEVICE_FEATURE_MIGRATION`, this isn't an allowed operation, but we allow `GET` and `PROBE`. For `VFIO_DEVICE_FEATURE_MIG_DEVICE_STATE` we allow all three operations, and since these are the only supported features (apart from the DMA logging one which I haven't started implementing yet), we return `false` for anything else.

Reply:
I'm not sure what my question was here.